### DISCUSSION TITLE
pkg upgrade wont work

### DISCUSSION THREAD

**Discussion by prdrj** [termux/termux-app#2564] [Q&A] [answered]
I've done pkg upgrade and getting error: maintenance or down 
I've tried changing repo using termux-change-repo but it doesn't work
Here's a screenshot 
![Screenshot_20220204-094012_Termux](https://user-images.githubusercontent.com/92352131/152471013-dadfe2de-5d4b-4f0d-a8ca-e0dd0cf5429f.jpg)

**Comment by agnostic-apollo** [comment, marked as answer]
https://github.com/termux/termux-app#google-play-store-deprecated

https://github.com/termux/termux-packages/issues/6726
